packages/cadl-python/test/mock_api_tests/test_core.py

```diff
@@ -8,7 +8,7 @@
 
 @pytest.fixture
 def client():
-    with CoreClient(api_version="") as client:
+    with CoreClient() as client:
```

Comment:
the core tests are updated due to cadl-ranch changes in PR https://github.com/Azure/cadl-ranch/pull/210